lib/csv2avro/converter.rb

```diff
@@ -1,14 +1,17 @@
 require 'csv2avro/schema'
 require 'csv2avro/avro_writer'
+require 'log/event'
+require 'log/metric'
 require 'csv'
 
 class CSV2Avro
   class Converter
-    def initialize(reader, writer, bad_rows_writer, error_writer, options, schema: schema)
+    def initialize(reader, writer, bad_rows_writer, log_writer, input_path, options, schema: schema)
```

Comment:
`log_writer` is not necessary for this class (can use `CSV2Avro.log.info` instead). `input_path` is not necessary either, only `filename` to be able to include that information in the events.